lib/templates/updateContextNgsi2.json

```diff
@@ -0,0 +1,38 @@
+{
+  "type": "object",
+  "properties": {
+    "actionType": {
+      "type": "string",
+      "enum": ["append"]
```

Comment:
In the context of CPr forwarding (i.e. CB sending a `POST /v2/op/update` to IOTA), how this template is used? To set the expecations of the request received from CB?

Reply:
Yes, this template is used is https://github.com/telefonicaid/iotagent-node-lib/blob/master/lib/services/northBound/contextServer.js#738 to validate incoming requests received from the CB. As you can see in https://github.com/telefonicaid/iotagent-node-lib/blob/master/lib/services/common/genericMiddleware.js#117, this is based on [revalidator](https://www.npmjs.com/package/revalidator) module which basically is able to check the requests against this JSON schema,

Reply:
In that case, use "update" instead of "append". In NGSIv1 forwarding POST /v1/updateContext used UPDATE, so in NGSIv2 is should be "update".

Reply:
(To make it clearer I have added this to the "specification" in https://github.com/telefonicaid/fiware-orion/issues/3068#issuecomment-470005604)

Reply:
Fixed with 1ad3afb50c6a474150d68e5154ca4effa688948e